tests/test_metrics_old.py

```
@@ -0,0 +1,454 @@
+    def test_anomaly_model_fallback_activations(self):
+        """Test model fallback activations counter"""
+        ANOMALY_MODEL_FALLBACK_ACTIVATIONS.inc()
+        assert ANOMALY_MODEL_FALLBACK_ACTIVATIONS._value == 1
+
+
+class TestRecoveryMetrics:
+    """Test recovery metric operations"""
+
+    def test_recovery_attempts_counter(self):
+        """Test recovery attempts counter"""
+        RECOVERY_ATTEMPTS_TOTAL.inc()
```

Ruff (0.14.10)

Undefined name `RECOVERY_ATTEMPTS_TOTAL`

(F821)